django/core/management/commands/flush.py

```diff
@@ -12,7 +12,7 @@ class Command(BaseCommand):
         'Removes ALL DATA from the database, including data added during '
         'migrations. Does not achieve a "fresh install" state.'
     )
-    stealth_options = ('reset_sequences', 'allow_cascade', 'inhibit_post_migrate')
+    stealth_options = ('reset_sequences', 'allow_cascade', 'inhibit_post_migrate', 'only_nonempty_tables')
```

Comment:
Might be useful as a non-stealth option? Would need documentation then. Also I'd be tempted to make it default `True`.

Reply:
Help with docs would be appreciated

Reply:
Added docs for the only place I could find `flush`, but the wording for a default True / set to False option would be clunky eg: all_tables_including_nonempty_tables or do_not_skip_non_empty_tables